experiments/azure/cyclecloud/cpu/experiment/configs/laghos/laghos-32n-out.log

```diff
@@ -0,0 +1,90 @@
+Start time: 1724313930
```

Comment:
For all these output files, 1. are we using them as final results, and 2. can we distinguish them from final results? For example, this one stops at step 155.

Reply:
The files should be removed, and they are now.